Formula/kaiju.rb

```diff
@@ -16,13 +16,19 @@ class Kaiju < Formula
   depends_on "python@2" # for convert_mar_to_kaiju.py
 
   def install
-    # https://github.com/bioinformatics-centre/kaiju/issues/92
-    inreplace "util/gbk2faa.pl", "/usr/bin/perl -w", "/usr/bin/env perl"
-
     system "make", "-C", "src"
 
     # https://github.com/bioinformatics-centre/kaiju/issues/93
-    rm "bin/taxonlist.tsv"
+    mkdir "share"
+    mv "bin/kaiju-convertMAR.py", "share"
+    mv "bin/kaiju-taxonlistEuk.tsv", "share"
+
+    pkgshare.install Dir["share/*"]
```

Comment:
```suggestion
    pkgshare.install "bin/kaiju-convertMAR.py", "bin/kaiju-taxonlistEuk.tsv"
```
Why make the temporary `share` subdirectory?